src/queries/article/user/articles.ts

```diff
@@ -27,12 +29,14 @@ const resolver: UserToArticlesResolver = async (
     inRangeEnd: input.filter?.inRangeEnd,
   })
 
-  return connectionFromPromisedArray(
-    draftService.dataloader.loadMany(
+  const drafts = uniqBy(
```

Comment:
It won't work if duplicated articles are across two pages?

Reply:
still need `.distinctOn`